packages/swingset-liveslots/src/liveslots.js

```diff
@@ -1319,7 +1319,7 @@ function build(
 
   /**
    * @param {import('./types.js').VatDeliveryObject} delivery
-   * @returns {void | Promise<void>}
+   * @returns {void | Promise<unknown>}
```

Comment:
```suggestion
   * @returns {undefined | Promise<unknown>}
```

But I'm wondering if we could type the result promise a little better since it's coming from `startVat`. Btw I really don't like the maybe promise return here, but we really shouldn't make this an async function given how it's wrapped into a "unmetered" call which doesn't support async functions.